tests/resources/complex_stack/suite.xml

```diff
@@ -56,6 +56,49 @@
             </background>
         </field>
     </detail>
+    <detail id="actions-test-case-list">
+        <title>
+            <text>Case List</text>
+        </title>
+        <action relevant="true()">
+            <display>
+                <text>Jump to Menu 2 Form 1</text>
+            </display>
+            <stack>
+                <push>
+                    <command value="'m1-f0'"/>
+                    <datum id="calculated_data" value="'new'"/>
+                </push>
+            </stack>
+        </action>
+        <action relevant="false()">
+            <display>
+                <text>Jump to Menu 2 Form 1</text>
+            </display>
+            <stack>
+                <push>
+                    <command value="'m1-f0'"/>
+                    <datum id="calculated_data" value="'new'"/>
+                </push>
+            </stack>
+        </action>
```

Comment:
@phillipm Would be nice to add an action here that doesn't have a relevancy condition at all and make sure it gets included